docs/guides/getting-started-orchestrate-human-tasks.md

```diff
@@ -30,12 +30,14 @@ To create an automated process with [user tasks](/components/modeler/bpmn/user-t
    ![modeler example](./img/modeler-example.png)
 4. Use Web Modeler to design a BPMN flow with user tasks. Create a user task by dragging the task icon from the palette, or click the existing start event and the displayed task icon.
 5. Change the task type by clicking the wrench icon. Select **User Task**.
-   ![wrench icon example](./img/wrench-icon-example.png)
+   ![wrench icon example](./img/wrench-icon-example-2.png)
 6. Add a descriptive name using the properties panel. In this case, we've named ours `Decide what's for dinner`.
 7. Append an exclusive [gateway](/components/modeler/bpmn/gateways.md) to your user task by clicking on the user task `Decide what's for dinner` and clicking on the element you'd like to create next. In this case, we've selected the diamond icon to create a gateway.
 8. Create two sequence flows (represented by the arrows) from the gateway and two new user tasks based on what the user decides to eat. In this case, we've named ours `Prepare chicken` and `Prepare salad`.
 9. Sequence flows require expressions to access variables from the form we'll create below to determine what to eat for dinner. To add an [expression](/components/concepts/expressions.md), click on the sequence flow to view the properties panel, and open the **Condition** tab to insert a conditional expression.
-10. Attach an end event to the two user tasks.
+10. Our sequence flow will have the expressions of `= Meal = "Salad"`on one side and `= Meal = "Chicken"`on the other.
```

Comment:
This is a bit of a nitpick. Steps in a guide should be action-oriented. Why not include this with step 9?

Reply:
Or perhaps re-write this to be an action - "Verify the sequences flows have the following expressions"

Reply:
Agreed here.